CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- CHANGELOG.md
+- Prettier
+- VTEX eslint rules
+### Changed
+- Remove deprecated eslint rules
+- Update eslint version
+- Remove semicolons
```

Comment:
Are you going to release a new version? These chores are not necessarily relevant to the changelog 🤔

Reply:
I will  release a new version and since that this had no changelog, I figured I should add one.